### DISCUSSION TITLE
Where is the source code and build instructions for dangerzone patched version of conmon?

### DISCUSSION THREAD

**Discussion by DeltaEpsilon19498** [freedomofpress/dangerzone#766] [General]
I am on Linux Mint, which is based on Ubuntu Jammy.

If I want to build from source, according to https://github.com/freedomofpress/dangerzone/blob/main/BUILD.md, "The conmon version that Podman uses and Ubuntu Jammy ships, has a bug that gets triggered by Dangerzone. If you want to run Dangerzone from source, you are advised to install a patched conmon version. A simple way to do so is to enable our apt-tools-prod repo, just for the conmon package". 

I cannot find the source code for this patched version. Nor can I find build instructions for it. And how would I get building dangerzone from source to use the patched version, as opposed to the version that my distro provides? 

I can find the binary version of conmon at https://packages.freedom.press/apt-tools-prod/pool/main/c/conmon/. However, in light of the recent xz vulnerability, some paranoia might be warranted, and I would prefer to build from source instead of trusting a binary. So where is the source code of the patched version and the build instructions for it?

Thank you for your help.

**Comment by apyrgio** [comment]
What a ride this `xz` situation has been...

I agree, some paranoia is warranted, and I hadn't realized that we don't point to the source. The patched `conmon` source is here: https://github.com/freedomofpress/maint-dangerzone-conmon/tree/ubuntu/jammy. The instructions on how to build it are here: https://github.com/freedomofpress/maint-dangerzone-conmon/tree/ubuntu/jammy/fpf

Hope that helps :slightly_smiling_face:

**Reply by DeltaEpsilon19498** [reply]
Thank you.

**Reply by nguyenquanvan** [reply]
> Thank you.
